src/main/java/com/cascadebot/cascadebot/commandmeta/ICommandExecutable.java

```diff
@@ -16,8 +19,19 @@
 
     CascadePermission getPermission();
 
+    String description();
+
     default boolean deleteMessages() {
         return true;
     }
 
+    /**
+     * Not when overriding this. the system automatically handles sub command arguments so you only need to define arguments here that are not defined their.
```

Comment:
`not when overriding this`?

Reply:
whops my bad

Reply:
What does it mean 👀

Reply:
Actually what is says is correct. I wanted to specify that sub commands are automatically handled so you don't need this method to handle arguments for sub commands (at least at the top sub command level)